index.js

```diff
@@ -0,0 +1,23 @@
+// fetchNews((newsData) => {
```

Comment:
If you require your other JS files here then the bundle.js will include them and you can just use the one file in your HTML (plus the bootstrap js)